package.json

```diff
@@ -61,9 +61,6 @@
       "eslint --fix"
     ]
   },
-  "prisma": {
-    "version": "a6a6ecba74cbaeee0756c312b1b0f2502b27e49c"
-  },
```

Comment:
Some, cleanup it seems it's not used since 7 months, since reorganizing the project into sub-packages.